src/pretix/plugins/sendmail/views.py

```diff
@@ -29,6 +29,12 @@ class SenderView(EventPermissionRequiredMixin, FormView):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['event'] = self.request.event
+        from_log_id = self.request.GET.get('from_log')
+        message = LogEntry.objects.get(id=from_log_id).display()
+        kwargs['message'] = message
+        self.form_class(initial=kwargs)
```

Comment:
Drop this line, it doesn't make a lot of sense. What you want to do instead is ``kwargs['initial'] = {'message': message}``. Django will later pass the ``kwargs`` to ``form_class`` in a different place.